manifests/addons/metrics-emitter.yml

```diff
@@ -0,0 +1,44 @@
+instance_groups:
+- name:      rmq-broker
+  jobs:
+  - name: loggregator_agent
+    release: loggregator-agent
+    consumes:
+      doppler:
+        from: doppler
+        deployment: (( grab meta.cf.deployment_name ))
+    properties:
+      loggregator:
+        tls:
+          ca_cert: (( vault meta.cf.exodus_path ":loggregator_ca" ))
+          agent:
+            cert: (( vault meta.cf.exodus_path ":loggregator_tls_agent_cert" ))
+            key: (( vault meta.cf.exodus_path ":loggregator_tls_agent_key" ))
+  - name: rabbitmq-metrics-emitter
+    release: rabbitmq-metrics-emitter
+    properties:
+      rabbitmq_metrics_emitter:
+        cloud_foundry:
+          api:    (( concat "https://api." meta.cf.system_domain ))
+          skip_ssl_validation: true
```

Comment:
Can skipping SSL validation be parameterized? We may actually use a trusted CA cert for the CF in many cases.